notebooks/4_visualize_forecast.ipynb

```diff
@@ -0,0 +1,1082 @@
+    "                first_intros_all[\"Destination\"] == destinations[i], \"TS\"\n",
+    "            ]\n",
+    "        )\n",
+    "    )\n",
+    "    axs[i].set_title(destinations[i])\n",
+    "plt.suptitle(\n",
+    "    f\"\"\"{run_name} \\n Year of First Introduction by Destination\"\"\", fontsize=18\n",
+    ")\n",
+    "\n",
+    "plt.savefig(f\"{results_dir}/figs/first_intro_by_destination.png\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Write plots for each country to file only:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Save separate plots for histograms of first intros by destination\n",
+    "for i in range(num_destinations):\n",
+    "    fig, ax = plt.subplots(1, figsize=(4, 3))\n",
+    "    fig.subplots_adjust(left=0.27, top=0.76, bottom=0.21)\n",
+    "    ax.hist(\n",
+    "        list(\n",
+    "            first_intros_all.loc[\n",
+    "                first_intros_all[\"Destination\"] == destinations[i], \"TS\"\n",
+    "            ]\n",
+    "        ),\n",
+    "        color=\"#31688e\",\n",
+    "    )\n",
+    "    ax.set_title(f\"\"\"{destinations[i]}\\nFirst Introduction Year\"\"\", fontsize=18, pad=14)\n",
+    "    ax.set_xlabel(\"year\", fontsize=18)\n",
+    "    ax.set_ylabel(\"% model runs\", fontsize=18)\n",
+    "    ax.set_xlim(left=2005, right=2030)\n",
+    "    ax.set_ylim(top=1000)\n",
+    "    y_vals = ax.get_yticks()\n",
+    "    ax.set_xticklabels([\"\", 2010, \"\", 2020, \"\", 2030], fontsize=16)\n",
+    "    ax.set_yticklabels(\n",
+    "        [\"{:3.0f}%\".format((x / 1000) * 100) for x in y_vals], fontsize=16\n",
+    "    )\n",
+    "    \n",
+    "    plt.savefig(f\"{results_dir}/figs/{destinations[i]}_first_intros.png\")\n",
+    "    plt.close()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "All introductions by country:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "destinations_all = list(set(org_dest_all[\"Destination\"]))\n",
+    "min_intro_prop = 0.5\n",
+    "destinations = []\n",
+    "for i in range(len(destinations_all)):\n",
+    "    if (\n",
+    "        len(org_dest_all.loc[org_dest_all[\"Destination\"] == destinations_all[i]])\n",
+    "        > num_runs * min_intro_prop\n",
+    "    ):\n",
+    "        destinations.append(destinations_all[i])\n",
+    "num_destinations = len(destinations)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Plot histograms of all intros by destination\n",
+    "fig, axs = plt.subplots(\n",
+    "    2, math.ceil(num_destinations / 2), sharey=True, sharex=True, figsize=(12, 6)\n",
+    ")\n",
+    "fig.subplots_adjust(hspace=0.35, wspace=0.15, top=0.82)\n",
```

Comment:
Add lines to hide last subplot if the number of destinations is odd:

if num_destinations % 2 != 0:
    axs[-1, -1].axis('off')